model/src/lib.rs

```diff
@@ -1477,6 +1527,25 @@ mod tests {
         assert!(relative.is_positive())
     }
 
+    #[test]
+    fn libp2p_placeholder_snapshot() {
+        let placeholder = Libp2pIdentity::placeholder().peer_id();
+        assert_eq!(
+            placeholder.to_string(),
+            "12D3KooWE1krEN7utXUFhLmLXLWPSuah5WQ7ChzUFetkMS7TRXsm"
```

Comment:
I'd extract this a public const - `NO_PEER_ID` or something.

Reply:
I've added a comment above: I'd extract it as a constant and add a doc test which shows that this value is derived from `0000..`

Reply:
> I've added a comment above: I'd extract it as a constant and add a doc test which shows that this value is derived from 0000..

I cannot easily add the test because `model` does not know about `deamon` crate where the seed is located. I could move the `seed` to `model` but I am not sure that is correct. For now I just removed the test.